internal/util/paramtable/component_param.go

```diff
@@ -813,6 +815,15 @@ func (p *queryCoordConfig) initCheckInterval() {
 	p.CheckInterval = time.Duration(checkInterval) * time.Millisecond
 }
 
+func (p *queryCoordConfig) initCheckRedundancyInterval() {
+	interval := p.Base.LoadWithDefault("queryCoord.checkRedundancyInterval", "60000")
+	checkInterval, err := strconv.ParseInt(interval, 10, 64)
+	if err != nil {
+		panic(err)
```

Comment:
Panic behavior is OK. But the message is vague and cannot explain why querycoord panics to users.